inventory/public/style.css

```diff
@@ -83,6 +84,51 @@ body{
 	font-size:24px;
 }
 
+#inventory-notifications {
+	position: absolute;
+	left: 330px;
+	top: 310px;
+	min-width: 250px;
+	min-height: 100px;
+	z-index: 1;
+}
+.inventory-notification {
+	display: none;  /* flex */
+	align-items: center;
+	padding: 10px 20px 10px 10px;
+	margin-bottom: 5px;
+	border: 1px solid;
+	background-image: url("/inventory-data/theme/newitem-background.png");
+	background-repeat: no-repeat;
+	background-size: cover;
+	opacity: 0;
+}
+.inventory-notification-modifier {
+	font-size: 32px;
+	margin-right: 10px;
+}
+.inventory-notification-icon {
+	width: 64px;
+	height: 64px;
+	margin-right: 10px;
+}
+.inventory-notification-info {
+	font-size: 32px;
+	margin-top: 5px;
+}
+.inventory-notification.visible{
+	position:relative;
+	display:flex;
+	animation-name:inventory-notification;
+	animation-duration:5s;
+}
+@keyframes inventory-notification{
+	0% {opacity:0;}
+	20% {opacity:1;}
```

Comment:
I played around with it and I think 5% would look good here. It feels snappier, or what do you think?

Reply:
Haven't played around so much with it, but 5% sounds nice!